specs/commandline/sign.md

```diff
@@ -37,8 +37,44 @@ Flags:
       --plugin-config strings    {key}={value} pairs that are passed as it is to a plugin, refer plugin's documentation to set appropriate values
       --signature-format string  signature envelope format, options: 'jws', 'cose' (default "jws")
   -u, --username string          username for registry operations (default to $NOTATION_USERNAME if not specified)
+      --use-image-manifest       optional for using OCI image manifest to store signatures in a registry
```

Comment:
should we use some user friendly name here instead of internal implementation? like ```--use-oci-1.1``` to reflect that if registry support oci1.1 add this flag ?

I know its not a good name but I'm open to suggestions

Reply:
@shizhMSFT @FeynmanZhou  any suggestions on the flag name?

Reply:
how about `--image-spec v1.1-image`

Reply:
+1. ORAS CLI uses `--image -spec <spec version>-<manifest type>`  for this kind of scenario.

Reply:
yeah ```--image-spec``` looks good to me. 

@iamsamirzon would like to get your opinion here